### PR TITLE
Set size of spacer in expandAlignedLeading to 0

### PR DESCRIPTION
**Pull request by gordonbrander**
Fixes https://github.com/subconsciousnetwork/subconscious/issues/873

### CODE REVIEW

File: xcode/Subconscious/Shared/Components/Common/ExpandAlignedLeadingViewModifier.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ struct ExpandAlignLeadingViewModifier: ViewModifier {
     func body(content: Content) -> some View {
         HStack(spacing: 0) {
             content
-            Spacer()
+            Spacer(minLength: 0)
```
Comment (gordonbrander) [inline xcode/Subconscious/Shared/Components/Common/ExpandAlignedLeadingViewModifier.swift:16]: It turns out that the spacing problem was not caused by the transclude failing to fill horizontal space. Rather, it was the spacer in this modifier taking up a minimum amount of space.

Setting the minimum to zero fixes the issue.

File: xcode/Subconscious/Shared/Components/Common/SubtextView.swift [hunk 2]
```diff
@@ -79,7 +79,6 @@ struct SubtextView: View {
                 }
             }
         }
-        .transition(.slide)
```
Comment (gordonbrander) [inline xcode/Subconscious/Shared/Components/Common/SubtextView.swift:82]: Unrelated change for issue discovered while troubleshooting the SubtextView. We should not be decorating SubtextView with transition(.slide). This is a general-purpose view, and slide is a very opinionated transition (slides left to right).

Comment (bfollington) [reply xcode/Subconscious/Shared/Components/Common/SubtextView.swift:82]: I'm not even sure why this is here in the first place, any transition should be on the container...

**Review by bfollington — APPROVED** [review]
LGTM!
